receiver/receivercreator/README.md

```diff
@@ -20,7 +20,16 @@ receiver_creator will watch for endpoints generated by these observers.
 A map of receiver names (e.g. `redis/1`) to a template for when and how to
 instantiate that receiver.
 
-**receivers.&lt;receiver_type/id&gt;.rule**
+**receivers.&lt;receiver_type/id&gt;.type** (_Required_)
```

Comment:
It is not clear what this line means. I am lost in the in the brackets, slashes and dots.

Reply:
Yeah it's kind of hard to describe the structure of the yaml when the value of the keys are variable.

```
receivers:
  receiver_creator/1:
    # Name of the extensions to watch for endpoints to start and stop.
    watch_observers: [k8s_observer]
    receivers:
      prometheus_simple:
```

In this example receiver_creator is the receiver_type and id is 1.